README.md

```diff
@@ -190,6 +190,32 @@ func TestSomething(t *testing.T) {
 }
 ```
 
+### Change value of mocked function argument
+It's common case when mocked function change argument value by its reference (i.e. from xorm lib ```engine.Find(&users)```) so the method returns only error. 
+Here is how you can change the value by using reflection:
+
+```go
+func TestSomething(t *testing.T) {
+  user := model.User{
+    Id: 1,
+    Login: "login",
+  }
+
+  mc := minimock.NewController(t)
+  defer mc.Finish()
+  xormMock := NewXormEngineInterfaceMock(mc)
+
+  xormMock.FindMock.Inspect(func(p1 interface{}, _ ...interface{}) {
+			// Trick to return new value of argument p1 by its ref
+			ptr := reflect.ValueOf(p1).Elem().Addr().Interface().(*[]model.User)
```

Comment:
If we want to add an example of mutating a mocked function arguments, I'd prefer a simpler one that has no empty interfaces, variadic parameters, reflection, and several types from 3rdparty libraries.

Reply:
Would be great if someone can simplify the example

Reply:
First of all I agree with @deadok22. Secondly, what's the reasoning of changing parameters that probably come from the same test? Even if you for some reason needed this I don't think we should put it into the readme because it seems like an anti pattern.